WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -24,40 +24,40 @@ public partial class WalletCoinsViewModel : RoutableViewModel
 {
 	private readonly IObservable<Unit> _balanceChanged;
 	private readonly WalletViewModel _walletViewModel;
-	private ObservableAsPropertyHelper<bool> _isAnySelected = ObservableAsPropertyHelper<bool>.Default();
-	[AutoNotify] private FlatTreeDataGridSource<WalletCoinViewModel> _source = new(Enumerable.Empty<WalletCoinViewModel>());
+	[AutoNotify] private IObservable<bool> _isAnySelected = Observable.Return(false);
+
+	[AutoNotify]
+	private FlatTreeDataGridSource<WalletCoinViewModel> _source = new(Enumerable.Empty<WalletCoinViewModel>());
 
 	public WalletCoinsViewModel(WalletViewModel walletViewModel, IObservable<Unit> balanceChanged)
 	{
 		_walletViewModel = walletViewModel;
 		_balanceChanged = balanceChanged;
-        SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		SetupCancel(false, true, true);
 
-        NextCommand = CancelCommand;
+		NextCommand = CancelCommand;
 		SkipCommand = ReactiveCommand.CreateFromTask(OnSendCoins);
 	}
 
-	public bool IsAnySelected => _isAnySelected.Value;
-
 	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
 	{
 		var coins = CreateCoinsObservable(_balanceChanged)
-			.Publish();
+			.Replay()
+			.RefCount();
```

Comment:
What are these for?

Reply:
Explained here :) https://github.com/zkSNACKs/WalletWasabi/pull/8875#discussion_r940022349